cdptools/databases/cloud_firestore_database.py

```diff
@@ -77,20 +77,24 @@ def __init__(
         # With credentials:
         if credentials_path:
             self._initialize_creds_db(credentials_path, name)
+
+            # We fetch all tables in case some tables exist in target, but not in source
+            # Path returns a tuple, and root collection path is in first index
+            self._tables = [coll._path[0] for coll in self._root.collections()]
```

Comment:
Neat! Thanks for digging through the docs and finding this and also documenting :+1:

Reply:
It's kind of weird, the client's `collection(collection_path)` method is supposed to take in a tuple for the path, but I couldn't get it to work when directly passing in the tuples received from the `collections()` method. I think it has something to do with how they call split on the first index but maybe I wasn't using it right. But for our case this works fine. 

For example, when you use `coll_path` instead of `coll_path[0]` it fetches the minutes_item_file collection path as the tuple `('minutes_item_file',)` but doesn't properly find the collection when you pass that path back in to `self._root.collection(path)`.

Here's the documentation for reference: https://googleapis.dev/python/firestore/latest/_modules/google/cloud/firestore_v1/client.html#Client
https://googleapis.dev/python/firestore/latest/collection.html

Reply:
From digging through those docs for a bit and trying it out, seems like using the`.id` property is the shorthand for this? Not sure though. Tried it out and gives me the collection names as expected though. 🤷‍♀ 
```suggestion
            self._tables = [coll.id for coll in self._root.collections()]
```